prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/notification/NotificationWorkFlowTask.java

```diff
@@ -0,0 +1,103 @@
+package com.redhat.parodos.tasks.notification;
+
+import com.redhat.parodos.notification.sdk.api.ApiClient;
+import com.redhat.parodos.notification.sdk.api.ApiException;
+import com.redhat.parodos.notification.sdk.api.Configuration;
+import com.redhat.parodos.notification.sdk.api.NotificationMessageApi;
+import com.redhat.parodos.notification.sdk.model.NotificationMessageCreateRequestDTO;
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflow.task.BaseWorkFlowTask;
+import com.redhat.parodos.workflow.task.enums.WorkFlowTaskOutput;
+import com.redhat.parodos.workflow.task.parameter.WorkFlowTaskParameter;
+import com.redhat.parodos.workflow.task.parameter.WorkFlowTaskParameterType;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import com.redhat.parodos.workflows.work.DefaultWorkReport;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import org.springframework.http.HttpHeaders;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.*;
+
+@Slf4j
+public class NotificationWorkFlowTask extends BaseWorkFlowTask {
+
+	private final NotificationMessageApi apiInstance;
+
+	public NotificationWorkFlowTask(String basePath) {
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+		apiClient.setBasePath(basePath); // basePath = "http://localhost:8080"
+		this.apiInstance = new NotificationMessageApi(apiClient);
+	}
+
+	@Override
+	public @NonNull List<WorkFlowTaskParameter> getWorkFlowTaskParameters() {
+		LinkedList<WorkFlowTaskParameter> params = new LinkedList<>();
+		params.add(WorkFlowTaskParameter.builder().key("type").type(WorkFlowTaskParameterType.TEXT).optional(false)
+				.description("Message Type").build());
+		params.add(WorkFlowTaskParameter.builder().key("body").type(WorkFlowTaskParameterType.TEXT).optional(false)
+				.description("Message Body").build());
+		params.add(WorkFlowTaskParameter.builder().key("subject").type(WorkFlowTaskParameterType.TEXT).optional(false)
+				.description("Message Subject").build());
+		// TODO Add an option for List parameter
```

Comment:
Is this TODO something we need to finalize? Will this be done in this PR or should we open new jira issue?

Reply:
This is something we have already discussed about its requirement
As far as I know, @rgolangh works on changing this (see [this](https://redhat-internal.slack.com/archives/C04RFJ5E5UN/p1679500900254169) discussion)
@rgolangh do you have any update on this?

Reply:
not really cause I'm on something else atm but hope to get something soon